entries/QUnit.config.xml

```diff
@@ -35,6 +35,13 @@
 				<p>Prior to version 1.16.0, this property was called <code>module</code>.</p>
 			</desc>
 		</property>
+		<property name="seed" type="String" default="undefined">
+			<desc>
+				This property tells QUnit to run tests in a seeded-random order. The value provided will be used as the seed in a pseudo-random number generator to ensure that results are reproducible.
+				<p>Randomly ordering your tests can help identify non-atomic tests which either depend on a previous test or are leaking state to following tests. This is particularly beneficial in a development CI or post-commit process.</p>
+				<p>If <code>seed</code> is specified in the page's url parameters, but no value is specified, then QUnit will generate a random value to use as the seed. You can access this value from this property in order to reproduce any potential test failures.</p>
```

Comment:
I think the last sentence could be a little clearer. Maybe "You can access the value from this property and use it to repeat the same sequence in another run."